__docs__/wonder-blocks-modal/modal-footer.stories.tsx

```diff
@@ -151,171 +172,134 @@ export const Default: StoryComponentType = {
     ),
 };
 
-export const Simple: StoryComponentType = () => (
-    <ModalDialog aria-labelledby={"modal-id-1"} style={styles.dialog}>
-        <ModalPanel
-            content={
-                <>
-                    <Title id="modal-id-1">Modal Title</Title>
-                    <Strut size={Spacing.large_24} />
-                    {longBody}
-                </>
-            }
-            footer={
-                <ModalFooter>
-                    <View />
-                </ModalFooter>
-            }
-        />
-    </ModalDialog>
-);
-
-Simple.parameters = {
-    docs: {
-        description: {
-            story: `This is a basic footer. It contains an empty
-            \`<View>\`, so it is completely blank.`,
-        },
-    },
-};
-
-export const WithButton: StoryComponentType = () => (
-    <ModalDialog aria-labelledby={"modal-id-2"} style={styles.dialog}>
-        <ModalPanel
-            content={
-                <>
-                    <Title id="modal-id-2">Modal Title</Title>
-                    <Strut size={Spacing.large_24} />
-                    {longBody}
-                </>
-            }
-            footer={
-                <ModalFooter>
-                    <Button onClick={() => {}}>Submit</Button>
-                </ModalFooter>
-            }
-        />
-    </ModalDialog>
-);
-
-WithButton.parameters = {
-    docs: {
-        description: {
-            story: `This is a \`<ModalFooter>\` with a \`<Button>\`
-            as a child. No additional styling is needed, as the footer
-            already has the style \`{justifyContent: "flex-end"}\`.`,
-        },
-    },
-};
-
-export const WithThreeActions: StoryComponentType = () => {
-    const mobile = "@media (max-width: 1023px)";
-    const desktop = "@media (min-width: 1024px)";
-
-    const buttonStyle = {
-        [desktop]: {
-            marginRight: Spacing.medium_16,
-        },
-        [mobile]: {
-            marginBottom: Spacing.medium_16,
-        },
-    } as const;
-
-    const containerStyle = {
-        [desktop]: {
-            flexDirection: "row",
-            justifyContent: "flex-end",
-        },
-        [mobile]: {
-            flexDirection: "column-reverse",
-            width: "100%",
-        },
-    } as const;
-
-    return (
-        <ModalDialog aria-labelledby={"modal-id-3"} style={styles.dialog}>
+/**
+ * This is a `<ModalFooter>` with a `<Button>` as a child. No additional styling
+ * is needed, as the footer already has the style `{justifyContent:
```

Comment:
**Question**: `{justifyConent: "flex-end"}` is showing up as a separate block for me. Is that intentional?

![image](https://github.com/Khan/wonder-blocks/assets/13231763/61054510-342b-42ea-a23a-7eff6f3752e7)

Reply:
oh nice catch! I'll fix this.